src/components/VIcon/VIcon.vue

```diff
@@ -29,7 +29,10 @@ export default {
      * importing an SVG should give us the path to the file.
      */
     iconPath: {
-      type: String,
+      /**
+       * In `just` our icons get transformed to Vue components
+       */
+      type: process.env.NODE_ENV === 'test' ? Object : String,
```

Comment:
Thank you for explaining this!

Reply:
This comment should read `jest`, not `just`, right?

Reply:
Yup!